.docker/000-default.conf

```diff
@@ -10,6 +10,19 @@
     PassEnv TRUSTED_PROXIES
     PassEnv TRUSTED_HOSTS
 
+    SetEnvIf X-Forwarded-Proto https HTTPS=on
+    RequestHeader set X-Forwarded-Proto "https"
+    RequestHeader set X-Forwarded-For "%{REMOTE_ADDR}s"
```

Comment:
not sure about the trailing "s" here

Reply:
https://httpd.apache.org/docs/current/mod/mod_headers.html
says "e" is for ENV variables. "s" is for SSL variables only

Reply:
Yep, never used that one, just copy & pasted from @mplawner example. 
Someone with a reverse proxy should actually test if it works as expected.